src/fu-util.c

```diff
@@ -1815,7 +1849,13 @@ fu_util_downgrade (FuUtilPrivate *priv, gchar **values, GError **error)
 	g_signal_connect (priv->client, "device-changed",
 			  G_CALLBACK (fu_util_update_device_changed_cb), priv);
 	priv->flags |= FWUPD_INSTALL_FLAG_ALLOW_OLDER;
-	return fu_util_update_device_with_release (priv, dev, rel, error);
+	if (!fu_util_update_device_with_release (priv, dev, rel, error))
+		return FALSE;
+
+	/* TRANSLATORS: success message where we made the firmware on the
+	 * device older than it was before */
+	g_print ("%s\n", _("Successfully downgraded device"));
```

Comment:
Rather than device how about fu_device_get_name?

Reply:
The same here, it's all a bit repetitive if you include the name again:

    Downgrading Unifying Receiver…
    Idle…                    [***************************************]
    Downgrading Unifying Receiver…***********************************]
    Writing…                 [***************************************]
    Successfully downgraded device